player/command.c

```diff
@@ -2004,6 +2004,14 @@ static int get_track_entry(int item, int action, void *arg, void *ctx)
     struct replaygain_data rg = has_rg ? *track->stream->codec->replaygain_data
                                        : (struct replaygain_data){0};
 
+    struct mp_tags *tags = track->stream ? track->stream->tags : (struct mp_tags *){0};
```

Comment:
I think this would just do NULL pointer. You likely wanted
```suggestion
    struct mp_tags *tags = track->stream ? track->stream->tags : &(struct mp_tags){0};
```